HousingSearchApi/V1/Infrastructure/Extensions/ElasticSearchExtensions.cs

```diff
@@ -22,6 +22,7 @@ public static void ConfigureElasticSearch(this IServiceCollection services, ICon
 
             var connectionSettings =
                 new ConnectionSettings(pool)
+                    // .ServerCertificateValidationCallback((sender, certificate, chain, sslPolicyErrors) => url == "https://localhost:9200") // For local connection to dev
```

Comment:
I doubt anyone that needs this in the future will know to look here to find what they need.
Perhaps, make this be added on if the environment == local?